src/SFA.DAS.AssessorService.Data/StandardRepository.cs

```diff
@@ -667,5 +668,47 @@ private DataTable ConstructStandardOptionsDataTable(IEnumerable<StandardOption>
 
             return dataTable;
         }
+
+        public async Task<bool> GetCoronationEmblemForStandardReferenceAndVersion(string standardReference, string version)
+        {
+            var sql = @"SELECT [CoronationEmblem]
+                       FROM [Standards] 
+                       WHERE [IFateReferenceNumber] = @standardReference AND Version = @version";
+
+            var results = await _unitOfWork.Connection.QueryAsync<bool>(
+                sql,
+                param: new { standardReference, version },
+                transaction: _unitOfWork.Transaction);
+
+            return results.Single();
+        }
+
+        public async Task<bool> GetCoronationEmblemByStandardUId(string standardUId)
+        {
+            var sql = @"SELECT [CoronationEmblem]
+                       FROM [Standards] 
+                       WHERE [StandardUId] = @standardUId";
+
+            var results = await _unitOfWork.Connection.QueryAsync<bool>(
+                sql,
+                param: new { standardUId },
+                transaction: _unitOfWork.Transaction);
+
+            return results.Single();
```

Comment:
I've added this and used it in the StartCertificateHandler. At the start of the Record a Grade journey, new certificates were being generated using data from the wrong standard version, because certificate data for new certificates is retrieved from the latest standard version regardless of the actual version being graded.

With this commit, the value of CoronationEmblem will be retrieved from the correct standard version. I raised a [bug ](https://skillsfundingagency.atlassian.net/browse/QF-1633)to address the other fields.
@ChrisJWoodcock